auth-foundation/src/main/java/com/okta/authfoundation/client/internal/NetworkUtils.kt

```diff
@@ -106,7 +106,7 @@ internal suspend fun <T> OidcConfiguration.internalPerformRequest(
     currentCoroutineContext().ensureActive()
     return withContext(ioDispatcher) {
         try {
-            val okHttpResponse = executeRequest(request)
+            val okHttpResponse = executeRequest(request, returnVerbatimResponse = shouldAttemptJsonDeserialization)
```

Comment:
I'm not so sure about the `shouldAttemptJsonDeserialization` name either, but this looks like it's preexisting. Is this the API that's only used by IDX?

It might be nice to be be more explicit about what this is used for.